health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/InboxFilterWrapper.js

```
@@ -13,6 +13,25 @@ const InboxFilterWrapper = (props) => {
   const [filterValues, setFilterValues] = useState(
     { status: null, onRoadCondition: null, terrain: null, securityQ1: null, securityQ2: null, facilityId:null }
   );
+  const [tableHeight,setTableHeight]=useState(props?.tableHeight || "33rem");
+
+  const parseRem = (remString) => {
+    // Remove 'rem' from the string and convert it to a number
+    return parseFloat(remString.replace('rem', ''));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)